src/test/java/seedu/address/logic/commands/AddInventoryCommandTest.java

```diff
@@ -0,0 +1,262 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static seedu.address.testutil.Assert.assertThrows;
+
+import java.nio.file.Path;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.function.Predicate;
+
+import org.junit.jupiter.api.Test;
+
+import javafx.collections.ObservableList;
+import seedu.address.commons.core.GuiSettings;
+import seedu.address.commons.core.index.Index;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.ProjectDashboard;
+import seedu.address.model.ReadOnlyProjectDashboard;
+import seedu.address.model.ReadOnlyUserPrefs;
+import seedu.address.model.inventory.Inventory;
+import seedu.address.model.member.Member;
+import seedu.address.model.task.Task;
+import seedu.address.testutil.InventoryBuilder;
+
+public class AddInventoryCommandTest {
+    @Test
+    public void constructor_nullTask_throwsNullPointerException() {
+        assertThrows(NullPointerException.class, () -> new AddInventoryCommand(null,null));
+    }
+
+    @Test
+    public void execute_inventoryAcceptedByModel_addSuccessful() throws Exception {
+        AddInventoryCommandTest.ModelStubAcceptingInventoryAdded modelStub = new AddInventoryCommandTest.ModelStubAcceptingInventoryAdded();
+        Inventory validInventory = new InventoryBuilder().build();
+
+        CommandResult commandResult = new AddInventoryCommand(new Index(1),
+                validInventory.getName(), validInventory.getPrice()).execute(modelStub);
+
+        assertEquals(String.format(AddInventoryCommand.MESSAGE_SUCCESS, validInventory), commandResult.getFeedbackToUser());
+        assertEquals(Arrays.asList(validInventory), modelStub.inventoriesAdded);
+    }
+
+    @Test
+    public void execute_duplicateInventory_throwsCommandException() {
+        Inventory validInventory = new InventoryBuilder().build();
+        AddInventoryCommand addInventoryCommand = new AddInventoryCommand(new Index(1),
+                                                            validInventory.getName(), validInventory.getPrice());
+        AddInventoryCommandTest.ModelStub modelStub = new AddInventoryCommandTest
+                                                            .ModelStubWithInventory(validInventory);
+
+        assertThrows(CommandException.class, addInventoryCommand.MESSAGE_DUPLICATE_INVENTORY, () ->
+                addInventoryCommand.execute(modelStub));
+    }
+
+    @Test
+    public void equals() {
+        Inventory alice = new InventoryBuilder().withName("Alice").build();
```

Comment:
You need to update the sample data here, Alice and all that doesn't make sense for `Inventory`. You should probably create sample inventory objects for `Testing purposes, and store them in `src/test/java/seedu/address/testutil/TypicalTasksMembers.java`